.docker.test.py

```diff
@@ -26,6 +26,7 @@
         'PY3',
         'SED',
         'TEXT',
+        'ZIG'
```

Comment:
```suggestion
        'ZIG',
```

(to prevent double-line diff the next time a language is added)